uv.lock

```diff
@@ -526,26 +530,6 @@ wheels = [
     { url = "https://files.pythonhosted.org/packages/d2/81/e3073017a8f5c75169e79108eda209e6089e3f96c9f197d307cbda7df71c/compressed_tensors-0.11.0-py3-none-any.whl", hash = "sha256:e1cbc46e1ae032b7ceea915fe18c8d2de5a54d3a50a607969b6bdfe703b6cb83", size = 179951, upload-time = "2025-08-19T18:59:29.308Z" },
 ]
 
-[[package]]
-name = "cupy-cuda12x"
```

Comment:
nice!

Not sure how this was pulled in anyway- good to see it removed. Maybe this was from `ray[cgraph]`?